### PR TITLE
Quickstart: Run/Develop PyTorch in VSCode with Docker on Intel GPU

### PR DESCRIPTION
**Pull request by liu-shaojun**
### 1. Why the change?

This submission adds a quickstart: Run/Develop PyTorch in VSCode with Docker on Intel GPU

You can view here: http://10.239.44.83:8008/doc/LLM/DockerGuides/docker_run_pytorch_inference_in_vscode.html

### CODE REVIEW

File: docs/readthedocs/source/doc/LLM/DockerGuides/docker_run_pytorch_inference_in_vscode.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+For Windows, you will need to install wsl extension to to the WSL environment. Open the Extensions view in VSCode (you can use the shortcut `Ctrl+Shift+X`), then search for and install the `WSL` extension.
+
+Press F1 to bring up the Command Palette and type in `WSL: Connect to WSL Using Distro` and select it and then select a specific WSL distro `Ubuntu`
+
+![installwslextention-ezgif com-video-to-gif-converter (1)](https://github.com/liu-shaojun/ipex-llm/assets/61072813/f53b9d17-1719-442c-b08f-d0c369476dc0)
+
+
+## Launch Container
+
+Open the Terminal in VSCode (you can use the shortcut `Ctrl+Shift+``), then pull ipex-llm-xpu Docker Image:
```
Comment (glorysdj) [inline docs/readthedocs/source/doc/LLM/DockerGuides/docker_run_pytorch_inference_in_vscode.md:46]: what is the shortcut
it shows Ctrl+Shift+&#96;  in the doc

Comment (liu-shaojun) [reply docs/readthedocs/source/doc/LLM/DockerGuides/docker_run_pytorch_inference_in_vscode.md]: It should be `` Ctrl+Shift+` ``, it is to open the terminal in VSCode, already updated.

**Comment by shane-huang** [comment]
upload the images to bigdl-project.github.io. I can't see the figures now.

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**